content/doc/book/installing/index.adoc

```diff
@@ -1,79 +1,334 @@
+<3> Makes (i.e. "publishes") port 8080 of the `jenkinsci/blueocean` container to
+port 8080 on the host machine. The 1st number represents the port on the host
+while the 2nd represents the container's port. Therefore, if you specified `-p
+49001:8080` for this option, you would be accessing Jenkins on your host machine
+through port 49001.
+<4> Maps the `/var/jenkins_home` directory in the container to the Docker
+link:https://docs.docker.com/engine/admin/volumes/volumes/[volume] with the name
+`jenkins-data`. If this volume does not exist, then this `docker run` command
+will automatically create the volume for you. +
+*Notes:*
+This volume could also be created independently using the
+https://docs.docker.com/engine/reference/commandline/volume_create/[`docker
+volume create`] command: +
+`docker volume create jenkins-data` +
+Instead of mapping the `/var/jenkins_home` directory to a Docker volume, you
+could also map this directory to one on your machine's local file system. For
+example, specifying the option +
+`-v $HOME/jenkins:/var/jenkins_home` would map the container's
+`/var/jenkins_home` directory to the `jenkins` subdirectory within the `$HOME`
+directory on your local machine, which would typically be
+`/Users/<your-username>/jenkins` or `/home/<your-username>/jenkins`.
+<6> The `jenkinsci/blueocean` Docker image itself. If this image has not already
+been downloaded, then this `docker run` command will automtically download the
+image for you. Furthermore, if any updates to this image were published since
+you last ran this command, then running this command again will automatically
+download these published image updates for you. +
+*Note:* This Docker image could also be downloaded (or updated) independently
+using the https://docs.docker.com/engine/reference/commandline/pull/[`docker
+pull`] command: +
+`docker pull jenkinsci/blueocean`
+
++
+*Note:* If copying and pasting the command snippet above doesn't work, try
+copying and pasting this annotation-free version here:
++
+[source,bash]
+----
+docker run \
+  --rm \
+  -d \
+  -p 8080:8080 \
+  -v jenkins-data:/var/jenkins_home \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  jenkinsci/blueocean
+----
+
++
+** *On Windows:* +
+   The following command is effectively the same command shown for macOS/Linux
+   above but if you're using the Windows command prompt, replace the backslash
+   symbols `\` at the end of each line with carat symbols instead `^`, which
+   gives you:
++
+[source]
+----
+docker run ^
+  --rm ^
+  -d ^
+  -p 8080:8080 ^
+  -v jenkins-data:/var/jenkins_home ^
+  -v /var/run/docker.sock:/var/run/docker.sock ^
+  jenkinsci/blueocean
+----
+
++
+*Tip:* If you have some experience with Docker, you can add an option to the
+`docker run` command like `--name jenkins-blueocean`, which would give the
+`jenkinsci/blueocean` container the name "jenkins-blueocean". This would make it
+easier to refer to the container if, for instance you needed to access the
```

Comment:
No slow starts, give the reason up front.

Reply:
Done - re-written in both this guide and the inclusion for the tutorials.